olive/passes/onnx/conversion.py

```diff
@@ -190,33 +190,50 @@ def _export_pytorch_model(
 
         onnx_model = None
         if config["use_dynamo_exporter"]:
-            torch_version = torch.__version__
-            if version.parse(torch_version) < version.parse("2.2.0"):
+            # Take the "release" version so that dev builds like 2.5.0dev1234 are treated as 2.5.0
+            torch_version = version.parse(torch.__version__).release
+            # The "legacy dynamo" is the torch.onnx_dynamo_export API
+            legacy_dynamo_supported_version = version.parse("2.2.0").release
+            # The new "dynamo" api is torch.onnx.export with dynamo=True
+            dynamo_supported_version = version.parse("2.5.0").release
+            if torch_version < legacy_dynamo_supported_version:
                 raise ImportError(
                     f"torch.onnx.dynamo_export is not available for torch version {torch_version}. "
-                    "Please upgrade your torch version to 2.2.0 or above."
+                    "Please upgrade your torch version to 2.5.0 or above."
                 )
-            from torch._dynamo import config as dynamo_config
-            from torch.onnx import dynamo_export
+            elif legacy_dynamo_supported_version <= torch_version < dynamo_supported_version:
+                from torch._dynamo import config as dynamo_config
```

Comment:
since the if condition raises an error, I think we can make this an if condition and move the common lines 205-208 from the elif/else blocks.
Also line 218 seems unnecessary since that is the condition for the else block?

Reply:
Done. PTAL